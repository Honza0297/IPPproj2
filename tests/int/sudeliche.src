<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
<instruction order="1" opcode="LABEL">
<arg1 type="label">fun1</arg1>
</instruction>
<instruction order="2" opcode="CREATEFRAME"/>
<instruction order="3" opcode="PUSHFRAME"/>
<instruction order="4" opcode="DEFVAR">
<arg1 type="var">LF@a</arg1>
</instruction>
<instruction order="5" opcode="WRITE">
<arg1 type="string">Zadejte\032cislo:\010</arg1>
</instruction>
<instruction order="6" opcode="READ">
<arg1 type="var">LF@a</arg1>
<arg2 type="type">int</arg2>
</instruction>
<instruction order="7" opcode="LABEL">
<arg1 type="label">cyklus</arg1>
</instruction>
<instruction order="8" opcode="SUB">
<arg1 type="var">LF@a</arg1>
<arg2 type="var">LF@a</arg2>
<arg3 type="int">2</arg3>
</instruction>
<instruction order="9" opcode="JUMPIFEQ">
<arg1 type="label">sudy</arg1>
<arg2 type="var">LF@a</arg2>
<arg3 type="int">0</arg3>
</instruction>
<instruction order="10" opcode="JUMPIFEQ">
<arg1 type="label">lichy</arg1>
<arg2 type="var">LF@a</arg2>
<arg3 type="int">1</arg3>
</instruction>
<instruction order="11" opcode="JUMP">
<arg1 type="label">cyklus</arg1>
</instruction>
<instruction order="12" opcode="LABEL">
<arg1 type="label">sudy</arg1>
</instruction>
<instruction order="13" opcode="WRITE">
<arg1 type="string">Cislo\032je\032sude</arg1>
</instruction>
<instruction order="14" opcode="EXIT">
<arg1 type="int">0</arg1>
</instruction>
<instruction order="15" opcode="LABEL">
<arg1 type="label">lichy</arg1>
</instruction>
<instruction order="16" opcode="WRITE">
<arg1 type="string">Cislo\032je\032liche</arg1>
</instruction>
<instruction order="17" opcode="EXIT">
<arg1 type="int">0</arg1>
</instruction>
</program>