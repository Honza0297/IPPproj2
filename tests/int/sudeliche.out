Zadejte cislo:
Cislo je sude